x-pack/plugins/canvas/public/components/home/hooks/use_clone_workpad.ts

```diff
@@ -20,7 +20,7 @@ export const useCloneWorkpad = () => {
   return useCallback(
     async (workpadId: string) => {
       try {
-        let workpad = await workpadService.get(workpadId);
```

Comment:
It looks like you changed `workpadService.get` to use the SOC `resolve` API under the hood.

However, since the ID passed to this Clone Workpad function does not originate from a deep link, I think you should not use `resolve` here. See the [Dev Guide FAQ](https://www.elastic.co/guide/en/kibana/master/sharing-saved-objects.html#sharing-saved-objects-faq-resolve-instead-of-get) -- we want to avoid overuse of `resolve`.

So my recommendation would be to change the workpad service to expose a separate "resolve" HTTP route, and leave the existing "get" route unchanged.